tests/providers/amazon/aws/transfers/test_mysql_to_s3.py

```diff
@@ -26,16 +26,16 @@
 
 
 class TestMySqlToS3Operator(unittest.TestCase):
```

Comment:
We should rename this

Reply:
I keep the TestMySqlToS3Operator because it tests the `MySqlToS3Operator` with parameters that is not available on `SqlToS3Operator`. 
There’s a TestSqlToS3Operator on `tests/providers/amazon/aws/transfers/test_sql_to_s3.py`

Reply:
Ah OK, I misunderstood. In that case we should mark these tests with `ignore::DeprecationWarning` to suppress the deprecation warnings.